tools/ci/tc/tests/test_decision.py

```diff
@@ -1,9 +1,16 @@
 import mock
 import pytest
+import os
+import sys
 
-from tools.ci.tc import decision
 from six import iteritems
 
+here = os.path.dirname(__file__)
```

Comment:
IIUC, this can be removed once we delete the root-level `__init__.py`, which will no longer be required after we migrate to py3-only. If so, let's file an issue and put down a TODO here.

Reply:
I see it's in the RFC, which is probably enough.

Reply:
Yeah, I wasn't sure if I should scatter TODOs about.

I will own opening an issue after this PR merges, noting we should clean these (and the `__init__.py` files) up once we go Py3-only.